### PR TITLE
Fixed name for `Weapon_SMG_05_F` (MP5K) for EDEN item.

### PR DESCRIPTION
**Pull request by ColdEvul**
**When merged this pull request will:**
- change the name of classname `Weapon_SMG_05_F` to `MP5K` in the editor.

### CODE REVIEW

File: addons/realisticnames/CfgVehicles.hpp [hunk 1]
```diff
@@ -513,7 +513,11 @@ class CfgVehicles {
     class Weapon_SMG_02_F: Weapon_Base_F {
         displayName = CSTRING(SMG_02_Name);
     };
-
+    
+    class Weapon_SMG_05_F : Weapon_Base_F {
```
Comment (jonpas) [inline addons/realisticnames/CfgVehicles.hpp]: Classname and `:` is together in the rest of the file.

**Comment by ColdEvul** [comment]
Fixed `:` typo.

**Comment by bux578** [comment]
This could go into 3.6.2 aswell

**Comment by jonpas** [comment]
Merged in 492f075ac87d3c19e34416b4b2fcf62dad442492, thank you!

**Pull request closed without merging**